templates/_devices.html

```diff
@@ -1,4 +1,4 @@
-<script src="//cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.15/lodash.min.js" type="text/javascript"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.15/lodash.min.js" integrity="sha512-3oappXMVVac3Ge3OndW0WqpGTWx9jjRJA8SXin8RxmPfc8rg87b31FGy14WHG/ZMRISo9pBjezW9y00RYAEONA==" crossorigin="anonymous"></script>
```

Comment:
The docs say that you could just use `crossorigin` as a tag w/o a value, and that should be equivalent to `anonymous`
> Setting the attribute name to an empty value, like crossorigin or crossorigin="", is the same as anonymous.

But other than that lgtm :+1: 

See: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin#sect1

Reply:
TIL.
A field without a value is just confusion and an extra google search waiting to happen. Leaving it explicit makes the code self-explanatory.